lib/autokey/service.py

```diff
@@ -475,6 +480,9 @@ def __init__(self, mediator: IoMediator, app):
 
         self.engine = self.scope["engine"]
 
+    def clear_error_records(self):
+        self.error_records.clear()
+
     @threaded
     def execute(self, script: model.Script, buffer=''):
```

Comment:
Buffer is unused, right? I assume you don't want to remove it for deprecation reasons?

Reply:
You checked based on the diff shown here? Then GitHub’s diff rendering is at fault. It is used here in line 493:
https://github.com/autokey/autokey/blob/5fcbe73d35f4c41718c57dfda37a79a8441e29cb/lib/autokey/service.py#L486-L493

Reply:
Ah, will check again later

Reply:
Yup, my mistake